authors/apps/friends/serializers.py

```diff
@@ -24,12 +24,8 @@ class Meta:
             'id', 'username', "number_of_followers", "number_of_following"
             )
 
-    def get_number_of_followers(self, obj):
-        return len(
-            json.loads(json.dumps([u.username for u in obj.followers.all()]))
-            )
+    def get_number_of_followers(self, followers):
+        return followers
```

Comment:
Hmm.. your serializer doesn't look like it's doing anything now, just returning what was passed to it.

Reply:
Working on it.